src/index.js

```diff
@@ -1,2 +1,7 @@
-export {default as RandomForestClassifier} from './RandomForestClassifier';
-export {default as RandomForestRegression} from './RandomForestRegression';
+import {default as RandomForestClassifier} from './RandomForestClassifier';
+import {default as RandomForestRegression} from './RandomForestRegression';
+
+export default {
+    RandomForestClassifier,
+    RandomForestRegression
+};
```

Comment:
I had some problems using rollup because it only exports the first one, this one works for me